docs/includes/fundamentals/read-operations/ReadOperationsTest.php

```diff
@@ -164,4 +166,20 @@ public function arrayElemMatch(): void
         $this->assertNotNull($movies);
         $this->assertCount(2, $movies);
     }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testReadPreference(): void
```

Comment:
@GromNaN I'm getting this error for the test, not sure why: `MongoDB\Driver\Exception\ConnectionTimeoutException: No suitable servers found (`serverSelectionTryOnce` set). Topology type: ReplicaSetWithPrimary`

Reply:
Tests are running on a single-node replica. There is no read replica, that's why it can't find one in the time limit.

Reply:
ah okay can I just bypass the test?

Reply:
The failure was on the other method: `QueryBuilderTest::testReadPreference()`, I updated the value.